src/palindrome.rs

```diff
@@ -1,4 +1,5 @@
 // http://rosettacode.org/wiki/Palindrome_detection
+// http://rosettacode.org/wiki/Test_a_function
```

Comment:
"Test a function" just proposes to demonstrate how would you test a function, using Palindrome as example. We have a test and we can just show it here.